lua/wire/server/wirelib.lua

```diff
@@ -578,28 +577,18 @@ function WireLib.Link_Start(idx, ent, pos, iname, material, color, width)
 
 	local input = ent.Inputs[iname]
```

Comment:
Oh, whoops, I misread the comment as complaining about the name of the `Inputs` field, not the variable. Finding time to work on Wiremod means sleep deprivation usually :wink: